src/main/java/soramitsu/irohautils/balancer/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package soramitsu.irohautils.balancer.config;
+
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Configuration
+public class ApplicationConfiguration {
+
+    private final IrohaConfiguration irohaConfiguration;
+
+    public ApplicationConfiguration(IrohaConfiguration irohaConfiguration) {
+        this.irohaConfiguration = irohaConfiguration;
+    }
+
+    @Bean
+    public Map<String, IrohaAPI> irohaAPI() {
+
+        List<String> peers = irohaConfiguration.getPeers();
+        Map<String, IrohaAPI> irohaPeers = new ConcurrentHashMap<>(peers.size());
```

Comment:
Maybe it is better here to return Immutable map instead?

Reply:
in further you can manage peers and restart routes in runtime